internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -320,6 +308,56 @@ func (r *AgentServiceConfigReconciler) Reconcile(origCtx context.Context, req ct
 	return ctrl.Result{}, r.Status().Update(ctx, instance)
 }
 
+func (r *AgentServiceConfigReconciler) exposeIPXERoute(instance *aiv1beta1.AgentServiceConfig) bool {
+	var exposeIPXERouteString string
+	exposeIPXERouteString, ok := instance.ObjectMeta.GetAnnotations()[exposeIPXERouteAnnotation]
+	if !ok {
+		exposeIPXERouteString = "false"
+	}
```

Comment:
I would think we'd want this logic reversed.
iPXE as a feature won't work if we don't have these routes, right?

@filanov @avishayt @mhrivnak do we want http exposed by default so iPXE can work or do we want people to have to opt-in?

Reply:
This is going to stay opt-in.